spec/std/file_spec.cr

```diff
@@ -37,6 +37,13 @@ describe "File" do
     str.should eq("Hello World\n" * 20)
   end
 
+  {% if !flag?(:windows) %}
```

Comment:
Yeah, `/proc` is only pretty much guaranteed to be mounted on windows, but it *can* be mounted on most unixes.